pkgs/os-specific/linux/joycond/default.nix

```diff
@@ -25,6 +25,9 @@ stdenv.mkDerivation rec {
 
     substituteInPlace $out/etc/systemd/system/joycond.service --replace \
       "ExecStart=/usr/bin/joycond" "ExecStart=$out/bin/joycond"
+
+    substituteInPlace $out/etc/udev/rules.d/89-joycond.rules --replace \
+      "/bin/setfacl"  "${acl}/bin/setfacl"
```

Comment:
This is necessary because [89-joycond.rules](https://github.com/DanielOgorchock/joycond/blob/e31db38eeae14c63331ea8dae972e3873d7ff6fa/udev/89-joycond.rules) hardcodes `/bin/setfacl` [here](https://github.com/DanielOgorchock/joycond/blob/e31db38eeae14c63331ea8dae972e3873d7ff6fa/udev/89-joycond.rules#L3) and [here](https://github.com/DanielOgorchock/joycond/blob/e31db38eeae14c63331ea8dae972e3873d7ff6fa/udev/89-joycond.rules#L5)